packages/core/lib/commands/init/initSource/truffle-config.js

```diff
@@ -11,25 +11,31 @@
  * Hands-off deployment with Infura
  * --------------------------------
  *
- * Is your application complex and requires lots of transactions to deploy?
+ * Is your application complex and require lots of transactions to deploy?
```

Comment:
Y'know, figuring out whether this should be singular or plural would be a lot easier if this question was correct grammar in the first place... proper grammar here would be "Is your application complex and does it require..." but that sounds so wordy...

Maybe this should be "Do you have a complex application that requires lots of transactions to deploy?" or something that doesn't require combining clauses with an "and".

Reply:
Sorry for the poor grammar in the initial wording I wrote for this!

Reply:
> Do you have a complex application that requires lots of transactions to deploy?

I like it

Reply:
No worries! @gnidan. I agree with @cds-amal. I like this "Do you have a complex application that requires lots of transactions to deploy?" wording too. Simple grammar ;)